### ISSUE TITLE
Add shiny unit tests

### ISSUE THREAD

**Issue by jwildfire** [ASA-DIA-InteractiveSafetyGraphics/safetyGraphics#63]
Most likely using `shinytest` package. Intro docs are [here](https://rstudio.github.io/shinytest/articles/shinytest.html)

**Comment by jwildfire** [comment]
@pburnsdata to take a look at `shinytest`

**Comment by jwildfire** [comment]
Pilot Tests
1. Using the default data, settings are valid and chart is drawn with no user input.
2. Loading ADBDS.csv, no chart is drawn and settings are not valid.

**Comment by pburnsdata** [comment]
shinytest tests for both of the Pilot Tests listed above have been created and reside in inst/eDISH_app/tests.  
These can be executed via:
```
library(shinytest)
testApp("inst/eDISH_app", "defaultPath.R")
testApp("inst/eDISH_app", "invalidPath.R")
```

I did not see a way to easily interact with DOM elements so these tests currently check whether the settings are valid using a value and check if the chart is drawn using an image compare. The image compare does not seem ideal and will fail across different OSs. Other options exist like http://ropensci.github.io/RSelenium/articles/shinytesting.html that may be worth exploring.
These shinytests are not currently included in Travis see #95

**Comment by pburnsdata** [comment]
Closing since pilot tests are complete and we are leaving this alone for now. We can create new issues for further work here.
